Queues - Laura - Calculator

lmelgarejos opened the pull request:
# Calculator
Congratulations! You're submitting your assignment.

## Comprehension Questions

|  Question 	|  Answer 	|
|---	|---	|
|  Describe how you stored user input in your program?	| Using .gets I could store input into a variable, an array or a hash. |
|  How did you determine what operation to perform?	|  Creating methods for each one, and calling certain one depending on the user input.  |
|  Do you feel like you used consistent indentation throughout your code? 	|   I do.	|
|  If you had more time, what would you have added to or changed about the program? 	|  Finish the optional, but also I would like to show the results like fixnum if they are integers instead of floats.	|

lmelgarejos committed:
Create Laura_Calculator.rb

lmelgarejos committed:
Create laura_calculator.rb

kariabancroft commented:
# Calculator
## What We're Looking For

|  Feature 	|   Feedback	&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|
|---	|---	|
|  Takes in two numbers and an operator and performs the mathematical operation. 	|   Well Done	|
|	**Baseline**	|	|
|  Readable code with consistent indentation. 	|  SO close,  Line 25  has an end that shouldn't be indented	|
|   **Extras**	|   	|
|   Functionality	|   The way it's checking to make sure it's a number only allows whole numbers to be input. I can't input number with decimals. How else could you check for valid numbers? 	|
|   Methods	|   Good start with methods.  Something to keep in mind, when thinking about _when_ to use methods, is if you see some code being repeated, it's a great opportunity to create a method for it. For example, lines 3-6 and 11-14 are doing the exact some thing! 	|

lmelgarejos committed:
Update Laura_Calculator.rb

Hi! I am sorry for the delay. I forgot to update this before. I add one conditional in the two first 'until', which allows floats numbers.
